doc/user-guide.adoc

```diff
@@ -414,9 +453,9 @@ spec:
     resourceRef: my-binding
 ----
 
-After the Service Binding Operator processes the `my-binding` custom resource of the `ServiceBindingRequest` type, it creates a `Secret` object. The `Secret` resource contains binding information that the backing service provides. The Runtime Component Operator injects the `Secret` resource as environment variables into application pods.
+Similar to what was described above, after the Service Binding Operator processes the `my-binding` custom resource of the `ServiceBindingRequest` type, it creates a `Secret` object. The `Secret` resource contains binding information that the backing service provides. The Runtime Component Operator injects the `Secret` resource as environment variables into application pods.
```

Comment:
- Change "Similar to what was described above, after the Service Binding Operator processes..." to "After the Service Binding Operator, as defined previously, processes...".
- Change "the `Secret` resource as environment variables into " to "the `Secret` resource as an environment variable into ".